docs/src/content/docs/de/guides/route-data.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Routendaten
+description: Erfahre, wie das Seitendatenmodell von Starlight zum Rendern deiner Seiten verwendet wird und wie du es anpassen kannst.
+---
+
+import { Steps } from '@astrojs/starlight/components';
+
+Wenn Starlight eine Seite in deiner Dokumentation rendert, erstellt es zunächst ein Routendatenobjekt, das den Inhalt der Seite darstellt.
+In diesem Leitfaden wird erklärt, wie Routendaten erzeugt werden, wie du sie verwendest und wie du sie anpassen kannst, um das Standardverhalten von Starlight zu ändern.
+
+Eine vollständige Liste der verfügbaren Eigenschaften findest du in der [„Routendaten-Referenz“](/de/reference/route-data/).
+
+## Was sind Routendaten?
+
+Starlight-Routendaten sind ein Objekt, das alle Informationen enthält, die zum Rendern einer einzelnen Seite benötigt werden.
```

Comment:
Is it weird to say “Daten sind ein Objekt”? i.e. “plural is a singular? Just realised I technically did this in the English version too although there I definitely think of “Starlight route data” as a single entity even if “data” is technically a plural form, so basically it’s a collective noun in my brain.

I wonder if here a translation more like “Starlight route data are _contained in_ an object…” could work better?